Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -343,9 +343,8 @@ SkPaint GraphicsContextSkia::createStrokePaint(std::optional<Color> strokeColor)
         // FIXME: Implement stroke patterns.
         UNUSED_PARAM(strokePattern);
         notImplemented();
-    } else if (auto strokeGradient = state.strokeBrush().gradient()) {
-        // FIXME: Implement stroke gradients.
-        UNUSED_PARAM(strokeGradient);
+    } else if (auto strokeGradient = state.strokeBrush().gradient())
+        paint.setShader(strokeGradient->shader(state.alpha(), state.strokeBrush().gradientSpaceTransform()));
         notImplemented();
```

Comment:
I think this `notImplemented()` is supposed to be removed?

Reply:
Indeed